naga/src/back/spv/block.rs

```diff
@@ -2610,6 +2610,7 @@ impl BlockContext<'_> {
     /// composite_id, index)` to an instruction that extracts the `index`th
     /// entry of the value with ID `composite_id` and assigns it to the slot
     /// with id `result_id` (which must have type `result_type_id`).
+    #[expect(clippy::too_many_arguments)]
```

Comment:
Could you fixup [Implement dot4I8Packed and dot4U8Packed](https://github.com/gfx-rs/wgpu/pull/7494/commits/bc9f93ef29944b1f8e25e0d2a348902ec72d631f) with this change?

Reply:
Done.